### DISCUSSION TITLE
Display MPP 1-3 in the Live Data

### DISCUSSION THREAD

**Discussion by GalOR7** [solectrus/solectrus#2524] [Ideas]
Hi Georg, have seen that you worked on this idea already. I have two strings which I would like to monitor in live data. The webpage on the home v3 offers this info - so it is hopefully not too complicated to grab. Might be one way to display this as a fly out bubble when one have a long click done on the pv generation panel. But also a graphic comparision of both strings seems useful. I really appreciate the work you did. The system runs flawless on my rasberry - cool stuff  BR Peter

**Comment by ledermann** [comment]
It is always nice to hear from a happy SOLECTRUS user :)

The MPP measurements 1-3 are already stored in InfluxDB, only the presentation in the user interface is missing, see here:
https://github.com/solectrus/solectrus/issues/1756

Even though I can't yet say when it will be implemented, it is in the pipeline. I am therefore closing this discussion
